evalml/model_understanding/graphs.py

```diff
@@ -45,13 +45,8 @@ def confusion_matrix(y_true, y_predicted, normalize_method='true'):
     """
     y_true = _convert_to_woodwork_structure(y_true)
     y_predicted = _convert_to_woodwork_structure(y_predicted)
-    y_true = _convert_woodwork_types_wrapper(y_true.to_series())
-    y_predicted = _convert_woodwork_types_wrapper(y_predicted.to_series())
-    if isinstance(y_true, pd.Series):
```

Comment:
Removed, since `_convert_woodwork_types_wrapper` will always return pandas so no need for if statement